### ISSUE TITLE
Csla.Analyzers Warning

### ISSUE THREAD

**Issue by animjn** [MarimerLLC/cslaforum#885]

**Comment by rockfordlhotka** [comment]
@jasonbock, any ideas?

I've seen this too, but not consistently.

**Comment by JasonBock** [comment]
Nope. I just created a .NET Core console app, referenced CSLA 5.1.0, and the analyzers seem to be working as expected. Maybe the project type matters? @animjn what project are you making? Is it targeting .NET Core or .NET Framework, and what version?

**Comment by animjn** [comment]
Hi @JasonBock. I was referencing CLSA from a Class Library (.NET Standard). I received the warnings when I was updating to 5.1.0, however I don't reproduce it anymore. Thanks!

**Comment by rockfordlhotka** [comment]
@animjn I have noticed that this can happen when first updating from an older version, and closing and reopening Visual Studio seems to help. I suspect Visual Studio keeps some old info in memory even after the update?
